site/docs/components/ag-grid-theme/examples.mdx

```diff
@@ -0,0 +1,180 @@
+---
+title: AG Grid Theme
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<AGThemeProvider>
+<LivePreviewControls>
+<LivePreview componentName="ag-grid-theme" exampleName="CheckboxSelection" >
+
+### Checkbox selection
+
+Rows can be selected via checkboxes.
+
+</LivePreview>
+<LivePreview componentName="ag-grid-theme" exampleName="Coloration" >
+
+### Coloration
+
+Cells, rows, or columns can have color applied to them.
+
+</LivePreview>
+<LivePreview componentName="ag-grid-theme" exampleName="ColumnGroup" >
+
+### Column group
+
+Similar to row groups, column groups collapse columns behind a group name.
+
+</LivePreview>
+<LivePreview componentName="ag-grid-theme" exampleName="ColumnSpanning" >
+
+### Column spanning
+
+Rows can span multiple columns.
+
+</LivePreview>
+<LivePreview componentName="ag-grid-theme" exampleName="ContextMenu" >
+
+### Context menu
+
+The menu can be opened on the column to view details or filters.
+
+</LivePreview>
+<LivePreview componentName="ag-grid-theme" exampleName="CustomFilter" >
+
+### Custom filter
+
+Users can quickly filter table information with the controls at the top.
+
+</LivePreview>
+<LivePreview componentName="ag-grid-theme" exampleName="Default" >
+
+### Default
+
+Default look when applying Salt theme
+
+</LivePreview>
+<LivePreview componentName="ag-grid-theme" exampleName="DragRowOrder" >
+
+### Drag row order
+
+Users can drag rows to rearrange.
+
+</LivePreview>
+<LivePreview componentName="ag-grid-theme" exampleName="Coloration" >
+
+### Coloration
+
+Colors communicate further information to the user.
+
+</LivePreview>
+<LivePreview componentName="ag-grid-theme" exampleName="FloatingFilter" >
+
+### Floating filter
+
+The filter remains visible as the user scrolls.
+
+</LivePreview>
+<LivePreview componentName="ag-grid-theme" exampleName="HDCompact" >
+
+### HD Compact
+
+description
```

Comment:
HD Compact theme is available by applying the `ag-grid-salt-high-compact` classname.